examples/linear_model/plot_bayesianridge_curvefit.py

```diff
@@ -0,0 +1,82 @@
+"""
+============================================
+Curve Fitting with Bayesian Ridge Regression
+============================================
+
+Computes a Bayesian Ridge Regression of Sinusoids.
+
+See :ref:`bayesian_ridge_regression` for more information on the regressor.
+
+In general, when fitting a curve with a polynomial by Bayesian ridge
+regression (evidence approxymation), the selection of initial values of
```

Comment:
typo: approximation

Reply:
alpha and lambda are not hyperparameters, alpha_init and lambda_init are so to be more correct I would write: the selection of the initial values of the regularization parameters (alpha, lambda).